### ISSUE TITLE
about BSQ Loss

### ISSUE THREAD

**Issue by VJJJJJJ1** [lucidrains/vector-quantize-pytorch#199]
hi, I'm trying to use BSQ to train my model, and the curve of BSQ loss as follows. Is normal? I wonder that why the loss is negative at the beginning and then increases gradually? Looking forward to your reply!

![Image](https://github.com/user-attachments/assets/722f44bc-d16f-4d26-9f56-085420fae4ac)

**Comment by VJJJJJJ1** [comment]
@lucidrains @sekstini @MisterBourbaki @kifarid
